### ISSUE TITLE
Using Extras for Multiple Scenes

### ISSUE THREAD

**Issue by jocklover** [CodyBerenson/PGMA-Modernized#133]
I am struggling to get the Extras functionality (method 2 from guide) to work correctly. I am not getting the additional scenes to show up in Plex. I have tried a fiddling with the config to see if I missed something, but no luck. :(

_Master Folder_: (Dream Boy Bondage) - Austin Young- Daddy's Boy (2019)
_Level 1 Video_: (Dream Boy Bondage) - Austin Young- Daddy's Boy (2019) Scene 1 _(was Chapter)_
_Level 1 Folder_: "Scenes"
_Level 2 Videos_:
- Scene 2
- (Dream Boy Bondage) - Austin Young- Daddy's Boy (2019) Chapter 3
- (Dream Boy Bondage) - Austin Young- Daddy's Boy (2019) Scene 4
(This was me trying different methods.)

https://www.iafd.com/title.rme/title=austin+young%3a+daddy%27s+boy/year=2019/austin-young-daddy&apos;s-boy.htm

All I see is one Poster for the Title but within the Poster there are no Extras listed, no additional scenes showing. And the video that plays is only the Scene 1.

**Comment by JPH71** [comment]
The scenes should be suffixed like this

   - (Dream Boy Bondage) - Austin Young- Daddy's Boy (2019) Chapter 3
   *-scene*

You have to have Local Media ticked and topmost in the list....
I think all files can be in the same folder...

Jason xx

**Comment by CodyBerenson** [comment]
@jocklover  Good catch.  As Jason says, the most recent version of the agents now requires a - before the scene (the Plex standard functionality that the usage guide was written to did not).  

I'll update the usage guide.

**Comment by CodyBerenson** [comment]
@JPH71 Jason, he is referring to Method 2 (Page 33) in the usage guide, using Plex's standard functionality to place extras in a sub folder.  I'll test to see if this functionality still works.

It is true that the recent release broke Method 1 (Page 30) by now requiring a - before Scene vs. Plex's standard naming convention for stacking which the usage guide had been written to;  therefore I'll update the Usage Guide for Method 1.

**Comment by CodyBerenson** [comment]
@jocklover 
I just tested using *EXACTLY* your setup above with one slight change, and it worked perfectly.  For the file in the Level 1 folder, you have to use the new naming convention...i.e. - Scene 1:
(Dream Boy Bondage) - Austin Young- Daddy's Boy (2019) - Scene 1

It does not matter what you name the files in the level 2 "Scenes" folder .  The contents display as "Extras", underneath the actors and underneath the Reviews.

Please note, in the Plex guidance for local media assets, they suggest that Local Media Assets be checked last:

![image](https://user-images.githubusercontent.com/40373801/156888091-a5892e01-36bd-41b9-9882-9f70d2ccbeef.png)

**Comment by JPH71** [comment]
The stacking guidance you gave is correct.
There are other suffices to use like -behindthescenes and -extras
These then show up like like extras on the films title page...

**Comment by JPH71** [comment]
Cody just notified me that when he read the message the bullet point was
showing up as an asterisk....

**Comment by jocklover** [comment]
THANK YOU - we have success!
A few notes:
- I did use " - Scene 1" in Folder 1
- I did enable Local Media Assets
- **Disabled duration check** (with Plex app restart). I was not getting IAFD metadata, checked log, and it was looking for the full 83 minute title duration not the 17 minute scene duration.
- Also, kept the "extra" scene videos as "Chapter #" w/ no issue

This is a huge unlock for me as I never liked stacking the scenes and have a lot of titles like this. THANK YOU ALL AGAIN!
